src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -70,6 +73,7 @@ export default class PathwayMapperTable extends React.Component<
             IPathwayMapperTableColumnType.SCORE,
             IPathwayMapperTableColumnType.GENES,
         ],
+        columnsOverride: { [IPathwayMapperTableColumnType.SCORE]: {} },
```

Comment:
This should support overriding all columns, not just the `SCORE`. I know that we don't need it at this point, but it's confusing to provide an interface that works only for a single column.